thumbor/doctor.py

```diff
@@ -58,15 +58,35 @@ def get_options():
     }
 
 
-def header(msg, color=cf.yellow):
-    print(color(msg))
+def print_message(color, msg):
+    print(f"{color}{msg}")
```

Comment:
Now this is a nit: perhaps now we can get rid of this method. That means one less function call. One less function in the code... Would you see any benefit in maintaining it?

Reply:
done!